cmake/common.cmake

```
@@ -252,7 +252,8 @@ function(_spl_coverage_create_overall_report)
         add_custom_command(
             OUTPUT ${COV_OUT_VARIANT_HTML}
             COMMAND gcovr --root ${CMAKE_SOURCE_DIR} --add-tracefile \"${CMAKE_CURRENT_BINARY_DIR}/**/coverage.json\" --html --html-details --output ${COV_OUT_VARIANT_HTML}
-            DEPENDS coverage
+            DEPENDS ${GLOBAL_COMPONENTS_COVERAGE_JSON_LIST}
+            COMMENT "Generating coverage overall report ${OUTPUT}..."

```

```suggestion
            COMMENT "Generating overall code coverage report ${OUTPUT} ..."
```

Really? ${OUTPUT} is available here?